src/components/pages/EdcConnector/ConfigurationDetailsOverlay.tsx

```diff
@@ -62,92 +61,83 @@ const ConfigurationDetailsOverlay = ({
             padding: '0px 120px 40px 120px',
           }}
         >
-          <Typography
-            variant="body1"
-            sx={{
-              marginBottom: '10px',
-            }}
-          >
-            {t('content.edcconnector.configurationDetails.orderList.title')}
-          </Typography>
-          <Trans
-            values={{
-              env: t(getCentralIdp()),
-            }}
-          >
-            <Typography variant="body2" className="detailsBodyText">
+          <Typography variant="body2" className="detailsBodyText">
+            <b>
               {t(
-                'content.edcconnector.configurationDetails.orderList.centralAuth'
+                'content.edcconnector.configurationDetails.section.part1.title'
               )}
-            </Typography>
-          </Trans>
-
+            </b>
+            {t(
+              'content.edcconnector.configurationDetails.section.part1.description'
+            )}
+            <span
+              className="detailsBodyLink"
+            >
+              {data?.decentralIdentityManagementAuthUrl}
+            </span>
+          </Typography>
           <Typography variant="body2" className="detailsBodyText">
-            {t('content.edcconnector.configurationDetails.orderList.clientId')}{' '}
+            <b>
+              {t(
+                'content.edcconnector.configurationDetails.section.part2.title'
+              )}
+            </b>
+            {t(
+              'content.edcconnector.configurationDetails.section.part2.description1'
+            )}
             <span
               className="detailsBodyLink"
-              onClick={() =>
-                window.open(
-                  t(
-                    'content.edcconnector.configurationDetails.orderList.clientSecretLink'
-                  ),
-                  '_blank'
-                )
-              }
+              style={{
+                textDecoration: 'underline',
+              }}
+              onClick={() => window.open('/technicaluser', '_blank')}
```

Comment:
please use defined page names for redirects:
https://github.com/eclipse-tractusx/portal-frontend/blob/main/src/types/Constants.ts#L49